src/core/agent/services/signifyNotificationService.ts

```diff
@@ -151,6 +153,13 @@ class SignifyNotificationService extends AgentService {
   }
 
   async deleteNotificationRecordById(id: string): Promise<void> {
+    await this.props.signifyClient
```

Comment:
We shouldn't delete this. We mark it as read on KERIA but I want them to persist.